svgcheck/run.py

```diff
@@ -108,10 +108,16 @@ def main():
         wp.color_threshold = options.grey_level
 
     if len(args) < 1:
-        with tempfile.NamedTemporaryFile(delete_on_close=False, mode="w+b") as tmp_file:
+        # delete_on_close was introduced in Python 3.12
+        delete_on_close = {"delete_on_close": False} if sys.version_info >= (3, 12) else {}
```

Comment:
Suggest a named flag like `have_delete_on_close = sys.version_info >= (3, 12)` to keep the version test in a single spot.

Reply:
Fixed with https://github.com/ietf-tools/svgcheck/pull/56/commits/bc612c1cb453ee439d02c46498fe2f6eb75fdd5e